packages/react-select/src/Select.tsx

```diff
@@ -76,7 +77,7 @@ export interface Props<
   /** HTML ID of an element containing an error message related to the input**/
   'aria-errormessage'?: string;
   /** Indicate if the value entered in the field is invalid **/
-  'aria-invalid'?: boolean;
+  'aria-invalid'?: AriaInvalid;
```

Comment:
Instead of redefining `AriaInvalid`, can we do `AriaAttributes['aria-invalid']` here?

Reply:
Yeah, definitely. I'll make that change now

Reply:
Do you think it's worth updating the other aria attributes to pull from that type in that case?

Reply:
If you're up for that, that would be great! Thanks.

Reply:
Done ✅